src/mikro-orm.middleware.ts

```diff
@@ -1,6 +1,5 @@
 import { MikroORM, RequestContext } from '@mikro-orm/core';
-import type { NestMiddleware } from '@nestjs/common';
-import { Injectable } from '@nestjs/common';
+import { Injectable ,type  NestMiddleware  } from '@nestjs/common';
```

Comment:
this issue is on more places, if this is not failing the linter, we are missing some rule...

```suggestion
import { Injectable, type  NestMiddleware  } from '@nestjs/common';
```

Reply:
'여러 개의 공백 없음'을 설정해야 하는 것을 발견하고 모두 수정했습니다.

Reply:
Found that no-multi-spaces should be set. I fixed it all.